src/Umbraco.Core/Handlers/AuditNotificationsHandler.cs

```diff
@@ -82,7 +82,7 @@ private IUser CurrentPerformingUser
         get
         {
             IUser? identity = _backOfficeSecurityAccessor.BackOfficeSecurity?.CurrentUser;
-            IUser? user = identity == null ? null : _userService.GetUserById(Convert.ToInt32(identity.Id));
+            IUser? user = identity == null ? null : _userService.GetAsync(identity.Key).GetAwaiter().GetResult();
```

Comment:
Same as for `UserNotificationsHandler´ - the surrounding notification handler could be made async and we could be rid of `.GetAwaiter().GetResult();` here.

Also, I really do not like awaiting an async result within a `get` property (`CurrentPerformingUser`). It should be turned into a method. Ideally an async method, but that would require the notification handler be rewritten as async too.